mod_app/models/film_model.py

```diff
@@ -72,6 +72,8 @@ def get_absolute_url(self):
         max_length=100, blank=True, null=True
     )  # can use choices if preset
 
+    bfi_identifier = models.CharField(max_length=7, blank=True, null=True)
```

Comment:
How sure are you that this is only 7 characters? This wikidata entry for [BFI National Archive work ID](https://www.wikidata.org/wiki/Property:P2703) gives examples with 9 characters.

Reply:
It's what Aylin said 🤷 : my question followed by her answer in brackets
>BFI identifier – are these always 5 digits or are they of variable length (eg 5-10 digits) and are they always numbers or do they sometimes have letters in as well? (  6-7 numbers)

Reply:
Ok. Unless you're relying on the validation, personally I'd opt for allowing longer entries just in case.